include/sys/spa_impl.h

```diff
@@ -456,6 +456,9 @@ struct spa {
 	boolean_t	spa_waiters_cancel;	/* waiters should return */
 
 	char		*spa_compatibility;	/* compatibility file(s) */
+	uint64_t	spa_dedup_table_quota;	/* property DDT maximum size */
+	uint64_t	spa_ddt_dsize;		/* cached on-disk size of DDT */
```

Comment:
This name asks for unification IMHO. I see there already spa_dedup_dspace, so why not spa_dedup_dsize or something like that?  Or the other way around, I don't care.

Reply:
Good point.  Fixed